annotations/source/en/adt/union/union.md

```diff
@@ -275,6 +275,70 @@ all cases when using it:
       })
     });
 
+### Reacting to Many Errors with One Result
+### Reacting To Anything
+### Dealing All Error Scenarios
+### Dealing All Error Scenarios in One Fell Swoop
+### Logging Any Errors
+
+Handling tagged unions like the above example shows we can now handle errors
+in general, yet still retain the ability to react to each individual
+type of error.
+
+In some scenarios, we wish to react to any response, not caring which
+one it is. Examples include finding required HTTP headers for a request.
+Another is when calling noops for log and resource deallocation. For
+those situations, we can use the `any` type.
+
+Let's say we want to verify an _Authorization_ header is present and
+has an acceptable value. There are 3 acceptable values. Anything else
+is considered an error. The below is a Node Restify request example
+that defines the 3 acceptable header values, 
+
+    const AuthHeader = union('AuthHeader', {
+      None() { return { 'None' }; },
+      JWT() { return { 'JWT' }; },
+      Session() { return { 'Session' }; },
+      Unknown() { return { 'Unknown' }; }
+    });
+
+    function hasAcceptableAuthHeader(authHeader) {
+      return authHeader.matchWith({
+        Unknown: () => false,
+        None: ()  => true,
+        JWT: ()  => true,
+        Session: ()  => true
+      })
+    }
+
+    api.method(response => {
+      // ... parse AuthHeader from response
+      if(hasAcceptableAuthHeader(authHeader)) {
+        // ... proceed
+      } else {
+        // ... failure
+      }
+    });
+
+We need the `AuthHeader` union throughtout our code to model our data,
+but in this particular case, any positive match of an acceptable data
+type is ok. Instead of having to write 3 all matches that result in 
+the same outcome, we can use the `any` symbol:
```

Comment:
Ending this paragraph with `::` allows the following piece of code to be used as a test, and forces the docs to be kept in sync if anything in to implementation changes.

Something like the following is enough to turn this into a useful test:

```js
const { any } = require('folktale/adt/union');

function hasAcceptableAuthHeader(authHeader) {
  return authHeader.matchWith({
    Unknown: () => false,
    [any]: () => true
  });
}

hasAcceptableAuthHeader(AuthHeader.Unknown());
// ==> false

hasAcceptableAuthHeader(AuthHeader.JWT());
// ==> true
```